site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -3,3 +3,61 @@ api: runtime
 ---
 
 <!-- Intentionally blank -->
+
+## Examples
+
+Using a content script to inject an extension asset on a page.
```

Comment:
This needs a better intro, sorry. How is this script started? How do I add it to my extension? This is an example of `getURL` but you should open with that.

Reply:
No worries. The current copy is terse to a fault. 

I agree that we should express call out `getURL` and better describe the general use case. I'm more hesitant on the other points as I don't think it's appropriate to go into detail about how to use content scripts or how they are executed. That information is better conveyed by more targeted docs. That said, we can and should link to those resources.

Take a look at the updated copy and let me know what you think.